go/pkg/client/bytestream_test.go

```diff
@@ -0,0 +1,294 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"testing"
+
+	"google.golang.org/grpc"
+
+	bsgrpc "google.golang.org/genproto/googleapis/bytestream"
+	bspb "google.golang.org/genproto/googleapis/bytestream"
+)
+
+var (
+	logStreams    map[string]*logStream
```

Comment:
This can be a member of the `ByteStream` strucrt on

Reply:
Done.